src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,7 +1,76 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { getMovie } from '../../api';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
 import './FindMovie.scss';
 
-export const FindMovie: React.FC = () => {
+const MOVIE_DEFAULT = {
+  title: '',
+  description: '',
+  imgUrl: '',
+  imdbUrl: '',
+  imdbId: '',
+};
+
+type Props = {
+  onMovieAdd: (movies: Movie) => void,
+};
+
+export const FindMovie: React.FC<Props> = ({ onMovieAdd }) => {
+  const [title, setTitle] = useState('');
+  const [movie, setMovie] = useState(MOVIE_DEFAULT);
```

Comment:
```suggestion
  const [movie, setMovie] = useState<Movie | null>(null);
```
you could do it in this way

Reply:
Hello! Thank You)